path_role.go

```diff
@@ -119,12 +160,15 @@ type jwtRole struct {
 	Period time.Duration `json:"period"`
 
 	// Role binding properties
-	BoundAudiences              []string                      `json:"bound_audiences"`
-	BoundSubject                string                        `json:"bound_subject"`
-	BoundCIDRs                  []*sockaddr.SockAddrMarshaler `json:"bound_cidrs"`
-	UserClaim                   string                        `json:"user_claim"`
-	GroupsClaim                 string                        `json:"groups_claim"`
-	GroupsClaimDelimiterPattern string                        `json:"groups_claim_delimiter_pattern"`
```

Comment:
What are the repercussions of removing this?

Reply:
The same capability can be achieved by revising the GroupsClaim to use a JSONPointer. This is definitely a backwards incompatibility, but I discussed with Jeff and he felt the user base of this fairly new field would be tiny, and the conversion to a JSONPointer is straightforward. This will be prominently called out in release notes/docs.

The immediate impact of doing nothing if you're using `GroupsClaimDelimiterPattern` is that group assignments won't happen.

Reply:
Sounds fine to me